src/client/app/components/maps/MapViewComponent.tsx

```diff
@@ -2,436 +2,91 @@
-			buttonMessageId = 'edit';
-		}
-
-		let toggleButton;
-		if (hasToken()) {
-			toggleButton = <Button style={this.styleToggleBtn()} color='primary' onClick={this.toggleNameInput}>
-				<FormattedMessage id={buttonMessageId} />
-			</Button>;
-		} else {
-			toggleButton = <div />;
-		}
-
-		if (hasToken()) {
-			return (
-				<div>
-					{formattedName}
-					{toggleButton}
-				</div>
-			);
-		} else {
-			return (
-				<div>
-					{this.props.map.name}
-					{toggleButton}
-				</div>
-			);
-		}
-	}
-
-	private toggleNoteInput() {
-		if (this.state.noteFocus) {
-			const editedMap = {
-				...this.props.map,
-				note: this.state.noteInput
-			};
-			this.props.editMapDetails(editedMap);
-		}
-		this.setState({ noteFocus: !this.state.noteFocus });
-	}
-
-	private handleNoteChange(event: React.ChangeEvent<HTMLTextAreaElement>) {
-		this.setState({ noteInput: event.target.value });
-	}
-
-	private formatNote() {
-		let formattedNote;
-		let buttonMessageId;
-		if (this.state.noteFocus) {
-			// default value for autoFocus is true and for all attributes that would be set autoFocus={true}
-			formattedNote = <textarea id={'note'} autoFocus value={this.state.noteInput} onChange={event => this.handleNoteChange(event)} />;
-			buttonMessageId = 'update';
-		} else {
-			formattedNote = <div>{this.state.noteInput}</div>;
-			buttonMessageId = 'edit';
-		}
-
-		let toggleButton;
-		if (hasToken()) {
-			toggleButton = <Button style={this.styleToggleBtn()} color='primary' onClick={this.toggleNoteInput}>
-				<FormattedMessage id={buttonMessageId} />
-			</Button>;
-		} else {
-			toggleButton = <div />;
-		}
-
-		if (hasToken()) {
-			return (
-				<div>
-					{formattedNote}
-					{toggleButton}
-				</div>
-			);
-		} else {
-			return (
-				<div>
-					{this.props.map.note}
-					{toggleButton}
-				</div>
-			);
-		}
-	}
-
-	private styleCalibrated(): React.CSSProperties {
-		return { color: 'black' };
-	}
-
-	private styleNotCalibrated(): React.CSSProperties {
-		return { color: 'gray' };
-	}
-
-	private formatCalibrationStatus() {
-		let styleFn;
-		let messageID;
-		if (this.props.map.origin && this.props.map.opposite) {
-			styleFn = this.styleCalibrated;
-			messageID = 'map.is.calibrated';
-		} else {
-			styleFn = this.styleNotCalibrated;
-			messageID = 'map.is.not.calibrated';
-		}
-		return (
-			<span>
-				<span style={styleFn()}>
-					<FormattedMessage id={messageID} />
+			</div>
+			<div className="item-container">
+				<b><FormattedMessage id="map.circle.size" /></b> {props.map.circleSize}
+			</div>
+			<div className="item-container">
+				<b><FormattedMessage id="map.modified.date" /></b>
+				{moment.parseZone(props.map.modifiedDate, undefined, true).format('dddd, MMM DD, YYYY hh:mm a')}
+			</div>
+			<div className="item-container">
+				<b><FormattedMessage id="map.filename" /></b> {props.map.filename}
+			</div>
+			<div className="item-container">
+				<b><FormattedMessage id="note" /></b> {props.map.note}
```

Comment:
Note should be limited to 30 characters as in other modals.

Reply:
I am unclear on why this was resolved since my look at the code does not see this addressed. Am I missing something?

Reply:
I'm not sure why it's showing this old code here. In VSCode the latest commit does not have this code.

Reply:
I see it too so it seems okay now.